sweetest/src/main/java/com/mysugr/sweetest/framework/context/DependenciesTestContext.kt

```diff
@@ -13,8 +13,10 @@ class DependenciesTestContext {
     @Deprecated("Use \"provide\" instead.", replaceWith = ReplaceWith("provide"))
```

Comment:
this class could be made internal i think...

Reply:
it could when adding @PublishedApi to every method. They are accessed form the `BaseBuilder`.